src/general-redaction/rapidjsonRedactor.cpp

```diff
@@ -0,0 +1,239 @@
+#include "../../include/general-redaction/rapidjsonRedactor.hpp"
+
+RapidjsonRedactor::RapidjsonRedactor() {
+    indent = 0;
+}
+
+void RapidjsonRedactor::redactAllInstancesOfMemberByName(rapidjson::Value& value, std::string member, bool& success) {
+    if (value.IsObject()) {
+        while (value.HasMember(member.c_str())) {
+            value.RemoveMember(member.c_str());
+            success = true;
```

Comment:
What are we doing with this success value? It seems to be passed in, then set but never used or checked for anything.